### ISSUE TITLE
Add POWERLEVEL_9k_MODE='awesome-fontconfig'

### ISSUE THREAD

**Issue by jaebradley** [jaebradley/dotfiles#35]
POWERLEVEL9K_MODE='awesome-fontconfig'

before specifying `ZSH_THEME`

https://github.com/Powerlevel9k/powerlevel9k/wiki/Install-Instructions#option-2-use-a-programmer-font

**Comment by jaebradley** [comment]
This worked in terms of getting fonts to display properly!

https://github.com/zeit/hyper/issues/464#issuecomment-531529521

**Comment by jaebradley** [comment]
Need to also 

```bash
brew tap homebrew/cask-fonts
brew cask install font-hack-nerd-font
```

**Comment by jaebradley** [comment]
Closed by #50
